CMakeLists.txt

```diff
@@ -2630,7 +2629,7 @@ if(HID)
     endif()
     target_link_libraries(mixxx-lib PRIVATE mixxx-hidapi)
   else()
-    message(STATUS "Linking libhidapi dynamically")
+    message(STATUS "Linking system libhidapi")
```

Comment:
This message now differs from all other external library "Linking ..." messages. Either change all or none.

Reply:
Changing all is beyond the scope of this PR. This message is misleading. The system library may be a static library.

Reply:
The let's leave the message as is. It causes more harm to introduce inconsistencies.